### PR TITLE
Jcm/merge in develop/release/1.3.0

### PR DESCRIPTION
**Pull request by jcmincke**
## Type of change

- [ ] 🐞 Bug fix (non-breaking change which fixes an issue)
- [ ] 🛠 New feature (non-breaking change which adds functionality)
- [ ] ⚠️ Breaking change (fix or feature that would cause existing functionality to change)
- [ ] 🏭 Refactoring that does not change existing functionality but does improve things like code readability, structure etc
- [ ] 🔨 New or improved tests for existing code
- [ ] ⛑ git-flow chore (backport, hotfix, etc)

## Developer checklist

- [ ] I have read the [style guide](https://github.com/input-output-hk/cardano-sl/blob/develop/docs/style-guide.md) document, and my code follows the code style of this project.
- [ ] If my code deals with exceptions, it follows the [guidelines](https://github.com/input-output-hk/cardano-sl/blob/develop/docs/exceptions.md).
- [ ] I have updated any documentation accordingly, if needed. Documentation changes can be reflected in opening a PR on [cardanodocs.com](https://github.com/input-output-hk/cardanodocs.com), amending the inline [Haddock](https://www.haskell.org/haddock/) comments, any relevant README file or one of the document listed in the [docs](https://github.com/input-output-hk/cardano-sl/tree/develop/docs) directory.

## Testing checklist

- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Comment by darko-mijic** [comment]
Reviewing this now...

**Comment by KtorZ** [comment]
Wouldn't it be better to point to PR rather than YT tickets? Since the changelog is on the repo and has kind of a public purpose?

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,36 +1,33 @@
...
+The endpoint that provides information about the node (`/api/v1/node-info`) was expanded to include
+information about the node subscription status, which can be used to check if the Cardano node is 
+connected to the network or not. (CBR-186)
 
-- Better error message for missing charset (Wallet Backend - CBR-223).
+#### Improved error message for missing charset in API calls
+The generic message returned from Cardano node endpoint calls when the charset was not specified is replaced with 
+a more descriptive error message to help users of the API troubleshoot the problem. (Wallet Backend - CBR-223)
 
-- Create AVVM mnemonic page screenshot (CBR-281).
```
Comment (darko-mijic) [inline CHANGELOG.md]: This is not a feature so I have removed it

File: CHANGELOG.md [hunk 2]
```diff
@@ -42,33 +39,54 @@
...
 
-- Node doesn't reconnect to the network (CDEC-259).
+- Upgrade to GHC 8.2.2 (CBR-51).
 
-- Wallet balance shows wrong Ada amount. Transaction is irrelevant to given wallet (CO-256).
+- Fix AppVeyor hard limitation on Windows (CBR-268).
 
 - Fix tmux versions in demo-script (CO-295).
 
-- Cannot create a Wallet via API V1 Wallet API (CO-315).
```
Comment (darko-mijic) [inline CHANGELOG.md]: This was not an issue so I have removed it.

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,36 +1,33 @@
 # CHANGELOG
 
-## Cardano SL 1.3.0 (Mainnet)
```
Comment (darko-mijic) [inline CHANGELOG.md]: The same code is running mainnet, testnet and staging. No need for the "Mainnnet" qualification here.

**Pull request merged**